command/job_inspect.go

```diff
@@ -90,6 +101,14 @@ func (c *JobInspectCommand) Run(args []string) int {
 	}
 	args = flags.Args()
 
+	if hcl && json {
+		c.Ui.Error("can only use one of -hcl or -json")
+		return 1
+	}
+	if withVars && !hcl {
+		c.Ui.Error("can only use -with-vars with -hcl")
```

Comment:
Is there a reason to `return 1` in the hcl+json flag error, but not in the withVars-hcl flag error?

Reply:
Nope, that's just a miss on my part. Will fix.